### ISSUE TITLE
about the Palette

### ISSUE THREAD

**Issue by y-kl8** [alicranck/coco2voc#1]
I want change the 3 channels rgb png which after proccessing into the 1 channel png which can be used by softmax of tensorflow. So, I  need the palette of the this coco,do you know the palette?

**Comment by alicranck** [comment]
Hi, if you refer to the label output I use the matplotlib 'inferno' colormap. If not please elaborate some more about the problem so I can try and help you.

**Comment by y-kl8** [comment]
yeah,I know the it, I want to know the specific value of the color. In voc,we have:
[(0,0,0)
  #0=background
 ,(128,0,0),(0,128,0),(128,128,0),(0,0,128),(128,0,128)
  #1=aeroplane, 2=bicycle, 3=bird,4=boat,5=bottle 
,(0,128,128),(128,128,128),(64,0,0),(192,0,0),(64,128,0)
  #6=bus, 7=car, 8=cat, 9=chair, 10=cow
,(192,128,0),(64,0,128),(192,0,128),(64,128,128),(192,128,128)
  #11=diningtable, 12=dog, 13=horse, 14=motorbike, 15=person
,(0,64,0),(128,64,0),(0,192,0),(128,192,0),(0,64,128)]
  #16=potted plant, 17=sheep, 18=sofa, 19=train, 20=tv/monitor


how about the this coco?  Becasue the softmax loss layer of tensorflow only recognize one channel label, I need to process the labels of rgb.

**Comment by alicranck** [comment]
There are class labels, however they are not assigned with the proper class id of the coco dataset. I recommend using the .npz files that are saved during the conversion. These files contain object ids for each pixel in the image, with which you can recover the class labels using pycocotools (see https://github.com/cocodataset/cocoapi). I will look at changing the class labels produced by the script as they are pretty useless in hindsight :/

**Comment by alicranck** [comment]
I changed the class labels so that each pixel value is the class id of the object in that pixel. Hope it will be more useful now.
